docusaurus/docs/themes/transportation/roads.mdx

```diff
@@ -197,12 +174,16 @@ allowed from that segment into other
 segments segments.
 
 Every road segment has an *implied* set of allowed transitions
-defined by its [directionality](shape-connectivity#direction-and-directionality),
-physical connections to other segments, the directionality of the
-physically connected segments, as well as local rules, norms, and
-customs. (The Overture Transportation schema does not specify these
-implied transitions, which are left to the specific application to
-resolve.)
+defined by its [access restrictions](#access-restrictions) as well as
+local rules, norms, and customs. An example of a transition restriction
+implied by an access restriction, if the segment can only be used along
+the `forward` [heading](shape-connectivity#heading), then it is implied
+that no transitions are allowed to any connected segments if travelling
+along the `backward` heading. An example of a transition restriction
```

Comment:
Are we saying that if a road is one-way headed south, we would not specify any turn restrictions as if you were driving north on this segment? 

Or are we saying if a road is one-way headed south, then turn restrictions are implied on all turns from east<->west road segments that would result in heading north on the one-way segment heading south?

If the second is true, I would suggest rewording as (bolding to highlight differences): 
An example of a transition restriction **would be if** a segment can only be used along the `forward` [heading](shape-connectivity#heading), then it is implied that no transitions are allowed **from any connected segments that would result in traveling along the `backward` heading.**

Reply:
The first one. If it's one-way heading south (as implied by the access restrictions) then the implication is you shouldn't be driving northbound, so we won't put any turn restrictions relating to driving north.

Reply:
ok, the first one is how I understood what you wrote. I realize upon reflection that I had so thoroughly assumed that we weren't going to specify turn restrictions assuming someone was driving the wrong way down the road that it didn't need to be an example, which isn't good documentation practice on my part. I think this works, and I withdraw my suggestion.